FastSurferCNN/data_loader/loader.py

```diff
@@ -116,7 +115,7 @@ def get_dataloader(cfg, mode):
 
             logger.info(f"Loading {mode.capitalize()} data ... from {data_path}. Using torchio Aug")
 
-            dataset = dset.MultiScaleDataset(data_path, cfg, gaussian_noise, transform)
+            dataset = dset.MultiScaleDataset(data_path, cfg, transform)
```

Comment:
Again, during training gaussian noise is needed, not during inference only. So gaussian_noise should be added here again